packages/backend/src/helpers/generate-error-email.ts

```diff
@@ -0,0 +1,87 @@
+import { DateTime } from 'luxon'
+
+import appConfig from '@/config/app'
+import { createRedisClient, REDIS_DB_INDEX } from '@/config/redis'
+import { sendEmail } from '@/helpers/send-email'
+import Flow from '@/models/flow'
+
+const MAX_LENGTH = 80
+const redisClient = createRedisClient(REDIS_DB_INDEX.PIPE_ERRORS)
+
+function redirectFlowLink(flowId: string) {
+  return `${
+    appConfig.isDev ? appConfig.webAppUrl : appConfig.baseUrl
+  }/editor/${flowId}`
+}
+
+function truncateFlowName(flowName: string) {
+  return flowName.length > MAX_LENGTH
+    ? flowName.slice(0, MAX_LENGTH) + '...'
+    : flowName
+}
+
+export function createBodyErrorMessage(
+  flowName: string,
+  flowId: string,
+): string {
+  const currDateTime = DateTime.now().toFormat('MMM dd yyyy, HH:mm:ss a')
```

Comment:
HH is 24-hour hours, so you will end up printing 23:00:00 pm. do we need seconds?

Reply:
ok ill remove the seconds thanks